### PR TITLE
Add target filters

### PR DESCRIPTION
**Pull request by clairemcginty**
Adds support for two new params in the upstream Missinglink library: [targetSourcePackages and targetDestinationPackages](https://github.com/spotify/missinglink/pull/266). These params allow the user to specify package names (unlike the existing ignore* options) which will filter found conflicts down to _only_ those conflicts that match on package/subpackage name.

cc @julienrf as most recent contributor 😃

### CODE REVIEW

**Review by sideeffffect — COMMENTED** [review]
Looks fine to me overall. But you need to fix the formatting :wink:

**Review by julienrf — APPROVED** [review]

File: src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala
Comment (clairemcginty) [reply src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala]: good idea! I'll update.

**Comment by clairemcginty** [comment]
thanks @sideeffffect! Addressed your comments. I think I need maintainer approval to run the CI workflow again :)

**Pull request merged**
